docs/learn/epoch/index.md

```diff
@@ -59,6 +59,18 @@ If not all orders can be executed, e.g. because there is not enough capital avai
 
 ## The Solver mechanism
 
+### How an epoch is currently displayed 
+These are the different states of an epoch.  
+
+| Status | Case | Consequence |
+| -------- | -------- | -------- |
+| Ongoing | Minimum epoch duration (24h) has not completed  | Minimum duration is passed (Investors can invest, redeem) |
```

Comment:
- How about formatting the status label in bold? I.e. '**Ongoing**'
- Rather 'Minimum duration has passed'
- Rather '(Investors can invest and redeem)'